server/amp.css

```diff
@@ -0,0 +1,185 @@
+/*!
```

Comment:
It's worth opening an issue just to log your reasoning in deciding to use Writ :ok_hand: